### PR TITLE
Split Article Java Loops

### PR DESCRIPTION
**Pull request by Rafase282**
Splices the article into smaller articles.

### CODE REVIEW

File: Java-Loops-Types-Do-While.md [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+
+```java
+do
+{
+    // Statements
+}
+while (expression);
+```
+
+This kind of loop is particularly useful if you want your `Statements` to be executed atleast once, irrespective of what `expression` evaluates to. You want to do this if you are initializing a variable inside your loop and plan on using its value later.
```
Comment (theflametrooper) [inline Java-Loops-Types-Do-While.md]: ~~atleast~~ at least

**Comment by theflametrooper** [comment]
## LGTM :+1:

File: Java-Loops-Control-Statement-Break.md [hunk 2]
```diff
@@ -0,0 +1,57 @@
+# break
```
Comment (koustuvsinha) [inline Java-Loops-Control-Statement-Break.md]: A bit more descriptive headers are needed than just break. Maybe : Java : the **break** statement?

File: Java-Loops-Control-Statement-Continue.md [hunk 3]
```diff
@@ -0,0 +1,46 @@
+# continue
```
Comment (koustuvsinha) [inline Java-Loops-Control-Statement-Continue.md]: Same for here. descriptive headers needed

File: Java-Loops-Control-Statement-Break.md [hunk 4]
```diff
@@ -0,0 +1,57 @@
...
+    {
+        if (i == 5 && j == 5)
+        {
+            break;
+            // Will terminate Loop 2, but Loop 1 will keep going
+        }
+    }
+}
+```
+
```
Comment (alayek) [inline Java-Loops-Control-Statement-Break.md]: Will have to add REPL code snippet. But maybe not in this PR.

File: Java-Loops-Control-Statements.md [hunk 5]
```diff
@@ -0,0 +1,10 @@
+# Loop Control Statements
```
Comment (koustuvsinha) [inline Java-Loops-Control-Statements.md]: **Java** Loop Control Statements

**Comment by koustuvsinha** [comment]
@Rafase282 descriptive headers needed for each article else we lose context. other than that its ok

**Comment by alayek** [comment]
# LGTM :+1: 

@koustuvsinha might want to take a look.

**Comment by Rafase282** [comment]
@koustuvsinha yes I noticed that last night on the last one, but I got to change the titles on this one now and also fix the typos.

**Pull request merged**